src/main/java/com/kodilla/ecommercee/controller/ProductController.java

```diff
@@ -29,19 +29,19 @@ public ProductController(ProductService productService, ProductMapper productMap
         this.productMapper = productMapper;
     }
 
-    @GetMapping(value = "getProducts")
+    @GetMapping
     public List<ProductDto> getProducts() {
         return productMapper.mapToProductDtoList(productService.getProductList());
     }
 
-    @GetMapping(value = "getProduct/{id}")
+    @GetMapping(value = "{id}")
     public ProductDto getProduct(@PathVariable("id") Long productId) throws ProductNotFoundException {
```

Comment:
Jeżeli ProductNotFoundException jest wyjątkiem Runtime, to czy informacja "throws" nie jest nadmiarowa?

Reply:
Faktycznie jest.